backend/src/test/java/com/woowacourse/levellog/acceptance/MyInfoAcceptanceTest.java

```diff
@@ -139,4 +144,96 @@ void findAllFeedbackToMe() {
                         "feedbacks.feedback.speak", containsInAnyOrder("페퍼 speak 리뷰", "로마 speak 리뷰"),
                         "feedbacks.feedback.etc", containsInAnyOrder("페퍼 etc 리뷰", "로마 etc 리뷰"));
     }
+
+    /*
+     * Scenario: 내가 작성한 레벨로그 조회하기
+     *   given: 2개의 팀에서 각각 레벨로그를 작성했다.
+     *   when: 내가 작성한 레벨로그 조회를 요청한다.
+     *   then: 내가 작성한 레벨로그 (2개) 와 200 OK 상태코드를 응답받는다.
+     */
+    @Test
+    @DisplayName("내가 작성한 레벨로그 조회")
+    void findAllMyLevellogs() {
+        // given
+        // 멤버 생성
+        final RestAssuredResponse romaLoginResponse = login("로마");
+        final String romaToken = romaLoginResponse.getToken();
+        final Long romaId = romaLoginResponse.getMemberId();
+
+        final RestAssuredResponse hostLoginResponse = login("호스트");
+        final String hostToken = hostLoginResponse.getToken();
+
+        // 팀 생성
+        final TeamCreateDto teamCreateDto1 = new TeamCreateDto("잠실 제이슨조", "트랙룸", LocalDateTime.now().plusDays(3),
+                new ParticipantIdsDto(List.of(romaId)));
+        final TeamCreateDto teamCreateDto2 = new TeamCreateDto("잠실 브리조", "톱오브스윙방", LocalDateTime.now().plusDays(3),
+                new ParticipantIdsDto(List.of(romaId)));
+
+        final String teamId1 = post("/api/teams", hostToken, teamCreateDto1).getTeamId();
+        final String teamId2 = post("/api/teams", hostToken, teamCreateDto2).getTeamId();
+
+        // 레벨로그 생성
+        final LevellogDto levellogRequest1 = LevellogDto.from("레벨로그1 내용");
+        final LevellogDto levellogRequest2 = LevellogDto.from("레벨로그2 내용");
+
+        final String levellogId1 = post("/api/teams/" + teamId1 + "/levellogs", romaToken, levellogRequest1)
+                .getLevellogId();
+        final String levellogId2 = post("/api/teams/" + teamId2 + "/levellogs", romaToken, levellogRequest2)
+                .getLevellogId();
+
+        // when
+        final ValidatableResponse response = RestAssured.given(specification).log().all()
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + romaToken)
+                .filter(document("myinfo/findAllMyLevellogs"))
+                .when()
+                .get("/api/my-info/levellogs")
+                .then().log().all();
+
+        // then
+        response.statusCode(HttpStatus.OK.value())
+                .body("levellogs.content", contains("레벨로그1 내용", "레벨로그2 내용"));
```

Comment:
```suggestion
        response.statusCode(HttpStatus.OK.value())
                .body("levellogs.id", contains(Integer.parseInt(levellogId1), Integer.parseInt(levellogId2)))
                .body("levellogs.content", contains("레벨로그1 내용", "레벨로그2 내용"));
```

이 코드 있는 위에 levellogId1과 levellogId2를 선언만하고 사용하지 않는 것 같아요. then에서 검증하면서 사용하거나 변수 선언을 안해도 될 것 같아요 

이 코드의 밑에 있는 `findAllMyTeams()`와 통일성을 생각하면 변수 선언을 하지 않는게 좋아보여요!